docs/docsite/rst/dev_guide/testing_integration.rst

```diff
@@ -25,6 +25,22 @@ It provides tab completion in ``bash`` for the ``ansible-test`` test runner.
 Configuration
 =============
 
+ansible-test command
+--------------------
+
+The example below assumes ``test/runner`` is in your ``$PATH``. An easy way to achieve that
```

Comment:
```suggestion
The example below assumes ``bin`` is in your ``$PATH``. An easy way to achieve that
```

This was written before `ansible-test` was added to `bin`. At the time it was only available as `test/runner/ansible-test`, but is now also available from the preferred location of `bin/ansible-test`. Sourcing `hacking/env-setup` makes the one under `bin` available.